javascript/.env

```diff
@@ -1 +1,2 @@
-NODE_PATH=src
\ No newline at end of file
+NODE_PATH=src
+MONGODB_URI=mongodb+srv://readOnlyB:NGyb0MCUdaT8I8d4@cluster0.6bwgb.mongodb.net/slack?retryWrites=true&w=majority

```

Comment:
I believe this is secret information, should this really be checked into version control?